internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -124,7 +124,7 @@ func applyRoles(c *collector.Manifests, strategy *operatorsv1alpha1.StrategyDeta
 			perms = append(perms, perm)
 		}
 	}
-	strategy.Permissions = perms
+	strategy.Permissions = append(perms, strategy.Permissions...)
```

Comment:
It shows fine. Could we add a test case for its scenario in https://github.com/operator-framework/operator-sdk/blob/master/internal/generate/clusterserviceversion/clusterserviceversion_updaters_test.go ?

Reply:
Apparently tested there: generate/clusterserviceversion/clusterserviceversion_test.go, will add one or adapt existing